np.sh

```diff
@@ -17,8 +17,11 @@ fi
 
 trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
 
-node "$trashCli" node_modules &&
-npm install &&
+if test "$2" = '--keep'; then
```

Comment:
Oh, yeah, my mistake, didn't see this was using `sh`. And I guess the positioning might not be worth the trouble.